client/src/pages/users/Users.module.css

```diff
@@ -57,3 +57,14 @@
     gap: 12px;
   }
 }
+
+.searchContainer{
+  flex-wrap: wrap;
+}
```

Comment:
If we use the default styling behavior of `Group`, we can remove this custom CSS rule